tests/ci/run_pytest.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
there are two files pretty similar `run_pytest.py` and `runpytest.py`, is this one duplicated?

Reply:
I updated name based on a comment above. removed old file.